data/data-pipeline/data_pipeline/comparison_tool/src/utils.py

```diff
@@ -0,0 +1,330 @@
+import pandas as pd
```

Comment:
I know some of our backend is fairly untested, but adding tests here would be extremely helpful. And I think you've written these functions in a way that should be pretty easy to test! @esfoobar-usds can assist with tests if you have any confusion about the best directory for them, etc.

Reply:
@esfoobar-usds -- where's the right directory these days for tests? I think this relates to: https://github.com/usds/justice40-tool/issues/1076

Reply:
(this is also in #1460)

Reply:
#1076 talks about moving the score tests out of  `data_pipeline/etl/score/tests` and into `data_pipeline/tests`. All new tests should go into the latter.

Reply:
i added a ticket for this as an fyi